### PR TITLE
Feature/refactor mesh material resource

### PR DESCRIPTION
**Pull request by mariofv**

### CODE REVIEW

File: Engine/Module/ModuleActions.cpp [hunk 1]
```diff
@@ -190,3 +192,26 @@ void ModuleActions::ClearUndoRedoStacks()
 	ClearRedoStack();
 	ClearUndoStack();
 }
+
+void ModuleActions::HandleInput()
```
Comment (yalania) [inline Engine/Module/ModuleActions.cpp:196]: Why you added this? It is like you are adding feature/game_controller_input_handling branch here

File: Engine/ResourceManagement/Importer/ModelImporter.cpp [hunk 2]
```diff
@@ -111,20 +112,14 @@ void ModelImporter::ImportNode(const aiNode* root_node, const aiMatrix4x4& paren
...
-		{
-			Config texture;
-			texture.AddString(loaded_meshes_materials[i], "uid");
-			materials.push_back(texture);
-		}
-		node.AddChildrenConfig(materials, "Textures");
+		std::string material_file = LIBRARY_MATERIAL_FOLDER"/" + std::string(root_node->mName.data) + std::to_string(i) + ".matol";
+		material_importer->ImportMaterialFromMesh(scene, mesh_index, file_path, material_file.c_str());
+		node.AddString(material_file, "Material");
+
```
Comment (yalania) [inline Engine/ResourceManagement/Importer/ModelImporter.cpp:121]: Here you can copy the material in library files to be in assets files.

**Review by yalania — COMMENTED** [review]
Check that you are no adding the input refactor branch

**Comment by mariofv** [comment]
I'm closing this PR at the moment as it's code its gonna be changed in an incoming refactor to resources

**Pull request closed without merging**